frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -119,19 +124,9 @@ class Worksheet extends React.Component {
             brief: props.brief ? 1 : 0,
         };
 
-        $.ajax({
-            type: 'GET',
-            url:
-                '/rest/interpret/worksheet/' +
-                this.state.ws.uuid +
-                '?' +
-                queryString.stringify(queryParams),
-            // TODO: migrate to using main API
```

Comment:
Is this TODO no longer relevant? (I believe so, but just wanted to make sure)

Reply:
I think it's no longer relevant, the API call works fine as is.